scripts/NeuralNet/balldetector.py

```diff
@@ -21,5 +24,39 @@ def classify(self, img):
 		return True
 
 if __name__ == "__main__":
-	bd = BallDetector()				# Create Ball Detector class
-	print bd.classify("foo")	# Classify an image as ball-containing or not
\ No newline at end of file
+	# bd = BallDetector()				# Create Ball Detector class
+	# print bd.classify("foo")	# Classify an image as ball-containing or not
+
+	# TODO: Cleanup this code into something reusable
+	imageFolder = '../../images/training-imgs'
+
+	# Train the network with all of the images in the example folder,
+	# using their filenames ("no_boxes" or "boxes" as the tag)
+	n = Network()
+	n.trainWith(os.path.join(imageFolder), presentPrefix="ball")
+
```

Comment:
My understanding was that values from pretraining could be reused to initialize the neural net nodes at startup. Is that the eventual intent or are is the intent to run an initial training set through at startup?